File: Lista2/Exercícios_python/Casa/4.py
a = float(input("Digite o valor de A: "))
b = float(input("Digite o valor de B: "))
c = float(input("Digite o valor de C: "))

if a!=b and b!=c and c!=a:
    if a>b and a>c:
        if b>c:
            print("Maior número: ", a)
            print("Menor número: ", c)
            print("Número do meio: ", b)
        else:
            print("Maior número: ", a)
            print("Menor número: ", b)
            print("Número do meio: ", c)
    elif b>c and b>a:        
        if a>c:
            print("Maior número: ", b)
            print("Menor número: ", c)
            print("Número do meio: ", a)
        else:
            print("Maior número: ", b)
            print("Menor número: ", a)
            print("Número do meio: ", c)
    else:
        if a>b:
            print("Maior número: ", c)
            print("Menor número: ", b)
            print("Número do meio: ", a)
        else:
            print("Maior número: ", c)
            print("Menor número: ", a)
            print("Número do meio: ", b)
else:
    print("Há números repetidos")

File: Lista2/Exercícios_C/Casa/4.c
#include <stdio.h>
int main ()

{
    float a, b, c;
    printf("Digite o valor de A: ");
    scanf("%f", &a);
    printf("Digite o valor de B: ");
    scanf("%f", &b);
    printf("Digite o valor de C: ");
    scanf("%f", &c);

    if (a!=b && b!=c && c!=a)
    {
        if (a>b && a>c)
        {
            if (b>c)
            {
                printf("Maior número: %.2f\nMenor número: %.2f\nNúmero do meio: %.2f", a, c, b);
            }
            else
            {
                printf("Maior número: %.2f\nMenor número: %.2f\nNúmero do meio: %.2f", a, b, c);
            }
        }
        else if (b>c && b>a)
        {
            if (a>c)
            {
                printf("Maior número: %.2f\nMenor número: %.2f\nNúmero do meio: %.2f", b, c, a);
            }
            else
            {
                printf("Maior número: %.2f\nMenor número: %.2f\nNúmero do meio: %.2f", b, a, c);
            }
        }
        else
        {
            if (a>b)
            {
                printf("Maior número: %.2f\nMenor número: %.2f\nNúmero do meio: %.2f", c, b, a);
            }
            else
            {
                printf("Maior número: %.2f\nMenor número: %.2f\nNúmero do meio: %.2f", c, a, b);
            }
        }
    }
    else
    {
        printf("Há números repetidos");
    }
}

